src/Components/Choices/Choices.css

```diff
@@ -87,85 +108,103 @@
     opacity: 0%;
     transform: rotate(360deg) scale(0);
   }
-}
-
-.incorrect {
-  background-color: rgb(8, 189, 189);
+ }
+ 
+ .incorrect {
+  z-index: 2;
+  background-color: rgb(8, 159, 159);
   border-radius: 10px;
   border: none;
-  width: 22%;
-  animation: drop 2.5s;
+  width: 45%;
+  height: 35%;
+  transition: all .2s;
+  color: white;
   font-size: x-large;
-}
-
-@keyframes drop {
+  font-weight: bold;
+  animation: drop 2.5s;
+ }
+ 
+ 
+ @keyframes drop {
   0% {
-    background-color: rgb(8, 189, 189);
+    background-color: rgb(8, 159, 159);
   }
   40% {
-    background-color: rgb(8, 189, 189);
+    background-color: rgb(8, 159, 159);
   }
   50% {
-    transform: translate(0vw, -10vh) scale(1.5);
+    transform: translate(0vw, -15vh) scale(1.2);
```

Comment:
The flip is great, but I think it's a little too close to the 'correct' answer animation.  I think it was a little more clear when it just fell off the page sideways.  Is there a tweak to be made to make the two a bit more distinct? Maybe when you get it wrong it doesn't do a full flip?

Reply:
Good call!